assets/control-plane/service-monitor-etcd.yaml

```diff
@@ -2,22 +2,22 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
-    k8s-app: etcd
+    app.kubernetes.io/name: etcd
   name: etcd
   namespace: openshift-monitoring
 spec:
   endpoints:
   - interval: 30s
-    port: etcd-metrics
+    port: metrics
```

Comment:
This seems to be an unintended change

Reply:
Good catch! :+1: I fixed it in the next commit.